reflex/config.py

```diff
@@ -190,6 +190,11 @@ class Config:
     # The rxdeploy url.
     rxdeploy_url: Optional[str] = None
 
+    # The hosting service backend URL.
+    cp_backend_url: str = constants.Hosting.CP_BACKEND_URL
+    # The hosting service frontend URL.
+    cp_web_url: str = constants.Hosting.CP_WEB_URL
```

Comment:
what users are expected to configure these knobs?

Reply:
not really. Nikhil just mentioned this to me in person too. Since our docs are auto generated, these will show up there. Having them here is mainly for internally we can easily change the endpoint to staging and dev for testing instead of having a separate branch with code changes. I guess we could leave them or add comments there are internal use only. Users do not need to set them?